editor/editor_properties.cpp

```diff
@@ -2649,6 +2649,7 @@ void EditorPropertyResource::update_property() {
 	if (res == RES()) {
 		assign->set_icon(Ref<Texture>());
 		assign->set_text(TTR("[empty]"));
+		assign->set_tooltip(TTR(""));
```

Comment:
No need for `TTR()`, that's a macro for translatable strings, and `""` doesn't need translation :)

Reply:
Could you also squash commits together?

Reply:
Ah okay thanks :). Yeah sure. Will that require a new pull request?

Reply:
No, you can squash the commits in your local branch, and force push the changes to your remote to update the pull request.
See instructions on http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase

Reply:
Thank you. That should be done now